system-contracts/contracts/L1Messenger.sol

```diff
@@ -238,7 +238,7 @@ contract L1Messenger is IL1Messenger, ISystemContract {
         /// Check messages
         uint32 numberOfMessages = uint32(bytes4(_totalL2ToL1PubdataAndStateDiffs[calldataPtr:calldataPtr + 4]));
         calldataPtr += 4;
-        bytes32 reconstructedChainedMessagesHash;
+        bytes32 reconstructedChainedMessagesHash = bytes32(0);
```

Comment:
Why?

Reply:
for all these OZ suggested adding in the default values. i dont have a strong opinion on keeping or removing